account_invoice_date_due/tests/test_account_invoice_date_due.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime, timedelta
+
+from odoo import fields
+from odoo.tests import Form, common
+
+
+class TestAccountInvoiceDateDue(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(cls.env.context, tracking_disable=True, no_reset_password=True)
+        )
+        # Create new user allowed to change invoice due date
+        group = cls.env.ref("account_invoice_date_due.allow_to_change_due_date")
+        acc_group = cls.env.ref("account.group_account_manager")
+        # Loose dependency on stock to avoid perm issues.
+        # We don't really care about such permissions in this context!
+        # Eg:
+        # odoo.exceptions.AccessError:
+        # You are not allowed to access 'Stock Valuation Layer' (stock.valuation.layer) records.
+        stock_group = (
+            cls.env.ref("stock.group_stock_manager", False) or cls.env["res.groups"]
+        )
+        cls.user_w_access = cls.env["res.users"].create(
+            {
+                "name": "Test User w/ access",
+                "login": "user_w_access",
+                "email": "somebody@somewhere.com",
+                "groups_id": [(6, 0, (group + acc_group + stock_group).ids)],
+            }
+        )
+        # Create new user not allowed to change invoice due date
+        cls.user_wo_access = cls.env["res.users"].create(
+            {
+                "name": "Test User wo/ access",
+                "login": "user_wo_access",
+                "groups_id": [(6, 0, (acc_group + stock_group).ids)],
+            }
+        )
+        account100 = cls.env["account.account"].create(
+            {
+                "code": "100",
+                "name": "Account 100",
+                "user_type_id": cls.env.ref("account.data_account_type_receivable").id,
```

Comment:
AFAIR: the field has been replaced by `account_type` and in this case the value should be `asset_receivable`